CTFd/themes/admin/templates/modals/users/addresses.html

```diff
@@ -12,7 +12,11 @@
 			<tbody>
 			{% for addr in addrs %}
 				<tr>
-					<td class="text-center">{{ addr.ip }}</td>
+					<td class="text-center">
+						<a href="{{ url_for('admin.view') }}/users?field=ip&q={{ addr.ip }}">
```

Comment:
You should be able to do this entirely with a url_for call. Something like `url_for('admin.users', field='ip', q=addr.ip)`.